Assets/Scripts/Control&Input/PlayerMovement.cs

```diff
@@ -92,6 +93,7 @@ public class PlayerMovement : MonoBehaviour
 
     [SerializeField, ReadOnly]
     private PlayerState state = PlayerState.GROUNDED;
+    public bool StateIsAir => state == PlayerState.IN_AIR;
```

Comment:
perhaps just IsInAir?

Reply:
Name collision with the IsInAir method

Reply:
Ah. My thought here was just that we might want to keep state internals out and focus on the fact that this property just tells us whether the player is in the air... But this doesn't matter _that_ much, so do whatever.